apps/src/componentLibrary/typography/components/BodyOneText.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Typography, {VisualApproach} from './../index';
+
+interface BodyOneTextProps {
```

Comment:
Seems like most if not all components share these set props - maybe easier to extract this out to `index` and reuse?